defaults/main.yml

```diff
@@ -70,6 +70,13 @@ mysql_log: ""
 mysql_log_error: /var/log/mysql.err
 mysql_syslog_tag: mysql
 
+mysql_config_include_files: []
+# Full example:
```

Comment:
Sure - I assume you'll want the other examples in this cleaned up as well, but I'll leave that for another day.